packages/styles/src/components/list.scss

```diff
@@ -0,0 +1,54 @@
+@use './../themes/bootstrap/core' as *;
+@use '../tokens/components' as components;
+@use '../functions/tokens' as tokens;
+@use './../functions/icons';
+
+tokens.$default-map: components.$post-list;
+
+$sizing-icon: tokens.get('post-list-checks', 'sizing-icon');
+$padding-icon: tokens.get('post-list-checks', 'spacing-padding-icon');
+
+.list-checks {
+  list-style: none;
+  padding-left: 0;
+  padding-block: tokens.get('post-list-checks', 'spacing-padding-block-outer');
+  margin-bottom: 0;
+
+  > li {
+    position: relative;
+    padding-inline-start: calc(
+      #{tokens.get('post-list-checks', 'spacing-gap-inline-text'),
+      } + #{$sizing-icon}
+    );
+
+    &::before {
+      content: '';
+      position: absolute;
+      left: 0;
+      width: $sizing-icon;
+      height: $sizing-icon;
+      padding: $padding-icon;
+      background-repeat: no-repeat;
+      background-position-x: 50%;
+      background-position-y: center;
+      background-color: tokens.get('post-list-checks', 'color-icon-bg');
+      border-radius: 50%;
+      background-size: calc(#{$sizing-icon} - #{$padding-icon} * 2);
+      background-image: url('#{icons.get-colored-svg-url("2105", "white")}'); // 'white' should be changed with variable
+    }
+
+    @include media-breakpoint-down(sm) {
```

Comment:
Please replace this bootstrap mixin with it's counterpart from https://github.com/swisspost/design-system/blob/main/packages/styles/src/mixins/_media.scss

Reply:
It is replaced.